mlflow/utils/autologging_utils.py

```diff
@@ -600,8 +604,12 @@ def log_autolog_called(self, integration, call_args, call_kwargs):
         is invoked (e.g., when a user invokes `mlflow.sklearn.autolog()`)
 
         :param integration: The autologging integration for which `autolog()` was called.
-        :param config_args: The positional arguments passed to the `autolog()` call.
-        :param config_kwargs: The keyword arguments passed to the `autolog()` call.
+        :param call_args: **DEPRECATED** The positional arguments passed to the `autolog()` call.
+                          This field is empty in MLflow > 1.13.1; all arguments are passed in
+                          keyword form via `call_kwargs`.
```

Comment:
Should we raise a deprecation warnign if `call_args` is not an empty tuple?

Reply:
Added a warning!

Reply:
This might be an overkill but do we need a test?

Reply:
Done! Also added a case verifying that methods on the default logging impl don't throw.